### DISCUSSION TITLE
Teknologi

### DISCUSSION THREAD

**Discussion by LilNiclas** [Kaptajnerne/EksamensProjekt#7] [General]
Dette forum er ment til diskussioner af teknologi faget/området

**Comment by LilNiclas** [comment]
@lakruzz 

Hej Lars, vi har et spørgsmål angående github actions/workflows. Er det tilladt at vi bruger nogen af de allerede konstrueret workflows lavet af Github Actions? Eksempelvis byg og test af projekt med maven, eller byg af Docker image.

**Reply by LilNiclas** [reply]
Over tid går vi ud fra at vi tilføjer mere og mere til vores workflows

**Reply by lakruzz** [reply]
Yes, selvfølgelig. God ide bar at starte med 'noget' og så udbygge det

_continuos improvement_ 👌
